lib/server.js

```diff
@@ -377,12 +377,18 @@ function unWindows(filePath) {
 // paths that the run.html.ejs file will contain.  The `rawSpecifierMap` is not
 // the entire importMap. It is a key/value map that may be the "imports" value
 // or an individual map inside of "scopes"[someScope].
-function reifyRawSpecifierMap(rawSpecifierMap) {
+function reifyRawSpecifierMap(rawSpecifierMap, options) {
   const concreteMap = {};
 
   for (const [key, value] of Object.entries(rawSpecifierMap)) {
     if (value.match(/^https?:\/\//)) {
       concreteMap[key] = value; // pass through unchanged
+    } else if (fs.existsSync(path.join(options.srcDir, value))) {
+      // Reuse existing __src__ path
+      concreteMap[key] = './' + unWindows(path.join('__src__', value));
+    } else if (fs.existsSync(path.join(options.specDir, value))) {
+      // Reuse existing __spec__ path
+      concreteMap[key] = './' + unWindows(path.join('__spec__', value));
```

Comment:
Is this needed?

Reply:
I wanted to support more scenarios for this. 
So it felt better to support file from these folders, too.
But I had no special use case in mind.

Reply:
You are right. 
Removed that while working on the tests.